source/includes/_tipser-elements.md

```diff
@@ -93,15 +93,21 @@ tipser.elements(posId: string, config?: TipserElementsConfig)
 
 ## `Store` Element
 
-Before you insert Store on your page, make sure there is at least one collection created in your store, otherwise no content will be rendered if your shop is empty.
+The Store is the best way to present a group of collections on a separate page. It is best to place it on a page where sufficient space is available, however, the script automatically blends in and adapts to the given space.
 
-Insert below HTML on your page in the place where you want the `Store` element to be rendered.
+Collections are created using the [Tipser Tools](#tipser-tools), or the Store Front Editor on [tipser.com](https://tipser.com).
```

Comment:
I don't think we should use the word Store Front Editor any longer.

Instead, I would write:
> or on "Manage store" dashboard on www.tipser.com (when logged in to a publisher account)